telegraf/telegraf.inputs.dct

```diff
@@ -0,0 +1,22 @@
+#######################  Docker/DCT services Metrics  ################################
+[[inputs.docker]]
+  endpoint = "unix:///var/run/docker.sock"
+  interval = "60s"
+  timeout = "10s"
+  namepass = [
+    "docker_container_cpu",
+    "docker_container_mem",
+    "docker_container_blkio",
+    "docker_container_status"
+  ]
+  docker_label_exclude = ["com.docker.compose.*"]
```

Comment:
We could also exclude the "resty" tags here as this info is not needed for graphing.  That would save almost 2KB from some of the metric entries.

`  docker_label_exclude = ["com.docker.compose.*", "resty*"]`